tests/FVTTests/testConfig.ts

```diff
@@ -0,0 +1,30 @@
+import { getDriver as getBrowserDriver, setApimlAuthTokenCookie  } from 'explorer-fvt-utilities';
+import { WebDriver } from 'selenium-webdriver';
+import { DEFAULT_SEARCH_FILTERS, loadPageWithFilterOptions } from './utilities';
+
+const {
+    ZOWE_USERNAME: USERNAME, ZOWE_PASSWORD: PASSWORD, SERVER_HOST_NAME, SERVER_HTTPS_PORT, TEST_BROWSER
+} = process.env;
+
+const BASE_URL = `https://${SERVER_HOST_NAME}:${SERVER_HTTPS_PORT}`;
+const BASE_URL_WITH_PATH = `${BASE_URL}/ui/v1/explorer-jes`;
+const FILTER_BASE_URL = `${BASE_URL_WITH_PATH}/#/`;
+const VIEWER_BASE_URL = `${BASE_URL_WITH_PATH}/#/viewer`;
+const loadUrlWithSearchFilters = loadPageWithFilterOptions(FILTER_BASE_URL, DEFAULT_SEARCH_FILTERS);
+const ZOSMF_JOB_NAME = 'IZUSVR1';
+
+export async function getDriver() {
```

Comment:
Can we add the return type to this function I think it will be Promise<WebDriver>

Reply:
whole utilities.ts has no return types, i believe if typescript able to derive type it don't complain, explicitly type not require for every function, its still type safe without explicit types